### PR TITLE
Add OCI support for unprivileged checks

### PR DESCRIPTION
**Pull request by lslebodn**
The support for ostree was removed from skopeo in fedora 31 and from podman in fedora 30.

### CODE REVIEW

**Review by lachmanfrantisek — COMMENTED** [review]
Looks good code-wise!

Should I play with that locally or is it ok to merge it and you will found possible errors yourself more easily?

I'll fix the packit config in a moment.

**Comment by lslebodn** [comment]
> Looks good code-wise!
> 
> Should I play with that locally or is it ok to merge it and you will found possible errors yourself more easily?

It is already covered by tests. (that was a way how I fixed bugs locally :-)
https://ci.centos.org/job/user-cont-colin-pr/123/console

I can rebase after merging #243

File: Dockerfile.tests
Comment (TomasTomecek) [inline Dockerfile.tests:1]: oh my, that's no longer supported

**Review by TomasTomecek — APPROVED** [review]
glad to see that the standard is OCI now

File: Dockerfile.tests
Comment (lslebodn) [reply Dockerfile.tests:1]: I know what is the reason why I created this pull request :-)

this change is explained in commit title
https://github.com/user-cont/colin/commit/6da0302fa64573312ea871aa4eed6067632e53eb.patch

f29 is the latest on which still reasonable support ostree (Or I would have to temporary switch to centos7)

I plan to drop support for ostree anyway and move to new distro as well.
But that will be done in different PR.

**Review by lachmanfrantisek — APPROVED** [review]
LGTM once rebased. Thanks!

**Pull request merged**
